packages/block-library/src/paragraph/edit.native.js

```diff
@@ -83,7 +82,9 @@ class ParagraphEdit extends Component {
 					tagName="p"
 					value={ content }
 					style={ {
-						...style,
+						// We don't want to pass style here from upper leyers, because it's corrupted
```

Comment:
It would be good if we can re-use the work we did for [the Inserter's SVGs case](https://github.com/WordPress/gutenberg/commit/2d474511dd100604421564ea3cfb745c300f7cc7#diff-34d04aad05a9f0ce1e1eb259845da0aeR24) where we take the concatenated string passed in the props and convert it to proper `style` format. Can we make a utility function out of that work and use it here?

Reply:
Hey @hypest @gziolo, I saw the solution for converting className to style, but when I look at the input (style string) and output (style object) it doesn't make sense for this case. 
This is what I get as a style`"custom-class-1 custom-class-2 has-background has-drop-cap has-large-font-size has-vivid-red-background-color"'` I think that I don't want that to be converted into style, as I didn't pass it. What do you think about it? I am open for suggestions.